include/openthread/platform/time.h

```diff
@@ -52,17 +52,34 @@ extern "C" {
  */
 
 /**
- * Get the current time (64bits width).
+ * Get the current time in microseconds referenced to a continuous monotonic
+ * local radio clock (64 bits width).
+ *
+ * The radio clock SHALL not wrap during the device's uptime. Implementations
+ * SHALL therefore identify and compensate for internal counter overflows. The
+ * clock does not have a defined epoch nor SHALL it introduce any continuous or
+ * discontinuous adjustments (e.g. leap seconds). Implementations SHALL
+ * compensate for any sleep times of the device.
+ *
+ * Implementations MAY choose to discipline the radio clock and compensate for
+ * sleep times by any means (e.g. by combining a high precision/low power RTC
+ * with a high resolution counter) as long as the exposed combined clock
+ * provides continuous monotonic microsecond resolution ticks within the
+ * accuracy limits announced by @ref otPlatTimeGetXtalAccuracy.
  *
  * @returns The current time in microseconds.
  *
  */
 uint64_t otPlatTimeGet(void);
 
 /**
- * Get the device's XTAL accuracy.
+ * Get the current estimated worst case accuracy (maximum ± deviation from the
+ * nominal frequency) of the local radio clock in units of PPM.
+ *
+ * Implementations MAY estimate this value based on current operating conditions
```

Comment:
There's some information missing here: in case the implementation does not estimate the value but returns a fixed value, this value MUST be the worst-case accuracy over all possible foreseen operating conditions (temperature, pressure, etc) of the implementation.

Reply:
Fixed.